app/src/main/java/com/himanshuvirmani/androidreactivearch/base/dependencyinjection/components/ApplicationComponent.java

```diff
@@ -41,5 +47,12 @@
 
   CacheManager cacheManager();
 
-  ApiManager apiManager();
+  //ApiManager apiManager();
+
+  // Added model specific methods
+  PostModel postModel();
```

Comment:
We can remove it from Application component and add model in each sub components which depends on Application Component,. For exam: We can add in PostComponent